build.sbt

```diff
@@ -1,17 +1,28 @@
-addCommandAlias(
-  "ci-test",
-  "+scalafmtCheckAll; +scalafmtSbtCheck; project-docs/mdoc; +coverage; +test; +coverageReport; +coverageAggregate"
-)
-addCommandAlias("ci-docs", "project-docs/mdoc; headerCreateAll")
+ThisBuild / scalaVersion := "2.13.2"
+ThisBuild / crossScalaVersions := Seq("2.12.11", "2.13.2")
+ThisBuild / organization := "com.47deg"
+
+addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; mdoc; testCovered")
+addCommandAlias("ci-docs", "github; mdoc; headerCreateAll")
+addCommandAlias("ci-publish", "github; ci-release")
 
 lazy val pbdirect = project
-  .in(file("."))
-  .settings(name := "pbdirect")
+  .settings(Compile / scalacOptions -= "-Xfatal-warnings")
```

Comment:
Is it possible to avoid moving all the files from the root into `pbdirect/`? It's not the end of the world, but it's slightly annoying and inconsistent with our other repos.

Could we leave these files where they are and put the documentation into a `docs` directory?

Reply:
This case is weird because we had two modules pending from the same `baseDirectory` (`pbdirect` and the root one), which can cause some conflicts.

Nonetheless is a pattern we also follow in other projects where the number of modules is not too big like [47degrees/fetch](https://github.com/47degrees/fetch), [47degrees/memeid](https://github.com/47degrees/memeid) or [47degrees/github4s](https://github.com/47degrees/github4s).